packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -80,6 +80,12 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
     required this.fadeoutOpacityAnimation,
     Color trackColor = const Color(0x00000000),
     Color trackBorderColor = const Color(0x00000000),
+    bool showButtons = false,
+    ScrollbarButtonsShape? buttonsShape,
+    bool isLeadingButtonHovered = false,
+    bool isLeadingButtonPressed = false,
+    bool isTrailingButtonHovered = false,
+    bool isTrailingButtonPressed = false,
```

Comment:
I'd like to find a way to not have so many parameters like this, the painter already has very many enumerated.